code/modules/arousal/organs/breasts.dm

```diff
@@ -17,8 +17,8 @@
 	arousal_verb = "Your breasts start feeling sensitive"
 	unarousal_verb = "Your breasts no longer feel sensitive"
 	orgasm_verb = "leaking"
-	fluid_transfer_factor = 0.5
-	var/static/list/breast_values = list("a" =  1, "b" = 2, "c" = 3, "d" = 4, "e" = 5, "f" = 6, "g" = 7, "h" = 8, "i" = 9, "j" = 10, "k" = 11, "l" = 12, "m" = 13, "n" = 14, "o" = 15, "huge" = 16, "flat" = 0)
+	fluid_transfer_factor = 1
+	var/static/list/breast_values = list("a" =  2, "b" = 4, "c" = 6, "d" = 8, "e" = 10, "f" = 6, "g" = 7, "h" = 8, "i" = 9, "j" = 10, "k" = 11, "l" = 12, "m" = 13, "n" = 14, "o" = 15, "huge" = 16, "flat" = 0)
```

Comment:
Why does E sized breasts give more milk than I sized breasts

Reply:
E is the actual cap